homeassistant/components/roborock/coordinator.py

```diff
@@ -75,9 +75,10 @@ async def verify_api(self) -> None:
                 # Right now this should never be called if the cloud api is the primary api,
                 # but in the future if it is, a new else should be added.
 
-    async def release(self) -> None:
+    def release(self) -> None:
         """Disconnect from API."""
-        await self.api.async_disconnect()
+        self.api.release()
```

Comment:
I see in code above that `api` might be the same as `cloud_api` if local API fails. Does calling `release` twice on the same object cause any issues?

Reply:
That's a really great question that I'll need to dive in and check and I can do that tomorrow

Reply:
Actually looked at it and it calls two functions:
Sync_disconnect which only runs if it is connected and task.stop() which cancel a asyncio task and according to asyncio docs, if it is already cancelled it will have no affect. 

So it should be good